sql/2019/06_Fonts/06_03.sql

```diff
@@ -0,0 +1,43 @@
+#standardSQL
+
+-- counts the font types (format)
+
+WITH
+  summary_requests AS (
+    SELECT * FROM `httparchive.summary_requests.2019_07_01_desktop`
+    UNION ALL
+    SELECT * FROM `httparchive.summary_requests.2019_07_01_mobile`
+  ),
+  processedTable AS (
+    SELECT
+      (
+        IF(STRPOS(mimeType, "font") > 0,
+          -- get the type of font from the content-type
+          REGEXP_REPLACE(
+            REGEXP_REPLACE(mimeType, r"^(font\/x\-)|(font\/font\-)|(font\/)|(application\/font\-)|(application\/x\-font\-)", ""),
+            r"^x\-",
+            ""
+          ),
+          -- hack: get the type of font from the file name
+          ARRAY_REVERSE(
+            SPLIT(
+              SPLIT(
+                ARRAY_REVERSE(
+                  SPLIT(url, "/")
+                )[OFFSET(0)],
+                "?"
+              )[OFFSET(0)],
+              "."
+            )
+          )[OFFSET(0)]
+        )
+      ) AS font,
+      *
+    FROM summary_requests WHERE type = 'font'
+  )
+SELECT
+  DISTINCT(font),
+  COUNT(font) AS count
```

Comment:
Again, prefer relative percents rather than absolute counts. This will help authors put the significance of the results in context.

Reply:
copy will change